### PR TITLE
Revised .NET Framework version DWORDs

### LINKED ISSUE #11516: Missing 394806 (4.6.2 on OSs other than Windows 10 Anniversary Update)
Based on the article [How to determine which versions and service pack levels of the Microsoft .NET Framework are installed](https://support.microsoft.com/en-us/help/318785/how-to-determine-which-versions-and-service-pack-levels-of-the-microso), 394806 is a valid value for 4.6.2. The table should probably be updated to reflect that.

I can confirm I get 394806 back from the registry on a Server 2012 R2 server with 4.6.2 installed. For a second I was worried that maybe I was running a preview version

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2115da38-e95e-46fc-1e2d-da88ef92c6cf
* Version Independent ID: a3790e8a-0654-851b-c991-ab7382025406
* Content: [How to: Determine which .NET Framework versions are installed](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed#ps_a)
* Content Source: [docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md](https://github.com/dotnet/docs/blob/master/docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md)
* Product: **dotnet-framework**
* GitHub Login: @rpetrusha
* Microsoft Alias: **ronpet**

### PR DESCRIPTION
**Pull request by rpetrusha**
## Revised .NET Framework version DWORDs

This restores the previous format of the **Release** DWORD table. Customer feedback about the change was uniformly negative.

Fixes #11516 

@vivmishra

### CODE REVIEW

File: docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md [hunk 1]
```diff
@@ -62,36 +62,36 @@ For information about detecting the installed updates for each version of the .N
...
-- To determine whether a minimum .NET Framework version is present, test whether the **Release** DWORD value found in the registry is *greater than or equal to* the value listed in the table. For example, if your application requires the .NET Framework 4.7 or later, you test for a minimum release key value of *460798*.
-
-- To test for multiple versions, begin with the latest .NET Framework version, and then test for each successive earlier version.
+The following table lists the value of the **Release** DWORD on individual operating systems for .NET Framework 4.5 and later versions.
 
 [!INCLUDE[Release key values note](~/includes/version-keys-note.md)]
 
 <a name="version_table"></a>
-
 |.NET Framework version|Value of the Release DWORD|
```
Comment (mairaw) [inline docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md]: this table is broken. you need the blank line for Markdig

File: docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md [hunk 2]
```diff
@@ -62,36 +62,37 @@ For information about detecting the installed updates for each version of the .N
...
-|.NET Framework 4.7|460798|
-|.NET Framework 4.7.1|461308|
-|.NET Framework 4.7.2|461808|
-
-For a complete table of release keys for the .NET Framework for specific Windows operating system versions, see [.NET Framework release keys and Windows operating system versions](release-keys-and-os-versions.md).
+|.NET Framework 4.5|All Windows operating systems: 378389|
+|.NET Framework 4.5.1|On Windows 8.1 and Windows Server 2012 R2: 378675<br />On all other Windows operating systems: 378758|
+|.NET Framework 4.5.2|All Windows operating systems: 379893|
+|.NET Framework 4.6|On Windows 10: 393295<br />On all other Windows operating systems: 393297|
+|.NET Framework 4.6.1|On Windows 10 November Update systems: 394254<br />On all other Windows operating systems: 394271|
```
Comment (mairaw) [inline docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md]: One of the confusions we heard from customers before is if all other Windows operating systems also includes the other Windows 10 versions? Can we make that more clear?

File: docs/framework/migration-guide/minimum-release-dword.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Minimum values of the Release DWORD for each .NET Framework version"
+ms.date: "04/02/2019"
+dev_langs:
```
Comment (mairaw) [inline docs/framework/migration-guide/minimum-release-dword.md]: you can remove dev_langs, You don't have any code here

**Review by mairaw — APPROVED** [review]
LGTM. Left a few comments.

File: docs/framework/migration-guide/minimum-release-dword.md (hunk 3 above)
Comment (rpetrusha) [reply docs/framework/migration-guide/minimum-release-dword.md]: There is code (imported from samples, on lines 108-109). Does the dev_langs attribute apply only to inline code, or does it apply to external code as well, @mairaw?

**Comment by rpetrusha** [comment]
@mairaw, can you take a quick look at this PR again. I've re-added content from #8307 that somehow was dropped.

File: docs/framework/migration-guide/minimum-release-dword.md (hunk 3 above)
Comment (mairaw) [reply docs/framework/migration-guide/minimum-release-dword.md]: External too but this article has 30 lines and no code from what I can see

File: docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md [hunk 4]
```diff
@@ -62,36 +62,37 @@ For information about detecting the installed updates for each version of the .N
...
+|.NET Framework 4.5.1|On Windows 8.1 and Windows Server 2012 R2: 378675<br />On all other Windows operating systems: 378758|
+|.NET Framework 4.5.2|All Windows operating systems: 379893|
+|.NET Framework 4.6|On Windows 10: 393295<br />On all other Windows operating systems: 393297|
+|.NET Framework 4.6.1|On Windows 10 November Update systems: 394254<br />On all other Windows operating systems (including Windows 10): 394271|
+|.NET Framework 4.6.2|On Windows 10 Anniversary Update and Windows Server 2016: 394802<br />On all other Windows operating systems (including other Windows 10 operating systems): 394806|
+|.NET Framework 4.7|On Windows 10 Creators Update: 460798<br />On all other Windows operating systems (including other Windows 10 operating systems): 460805| 
+|.NET Framework 4.7.1|On Windows 10 Fall Creators Update and Windows Server, version 1709: 461308<br/>On all other Windows operating systems (including other Windows 10 operating systems): 461310|
+|.NET Framework 4.7.2|On Windows 10 April 2018 Update and Windows Server, version 1803: 461808<br/>On Windows 10 October 2018 Update and all Windows operating systems other than Windows 10 April 2018 Update: 461814|
```
Comment (mairaw) [inline docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md]: would this read better?

```suggestion
|.NET Framework 4.7.2|On Windows 10 April 2018 Update and Windows Server, version 1803: 461808<br/>On all Windows operating systems other than Windows 10 April 2018 Update and Windows Server, version 1803: 461814|  
```

File: docs/framework/migration-guide/minimum-release-dword.md (hunk 3 above)
Comment (rpetrusha) [reply docs/framework/migration-guide/minimum-release-dword.md]: Argh. Of course. I was looking at the code in the how to determine what versions are installed article.

File: docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md (hunk 4 above)
Comment (rpetrusha) [reply docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md]: Yes, it does, @mairaw. Thanks!

**Pull request merged**